lib/tidewave/mcp/tools/source.ex

```diff
@@ -26,6 +28,33 @@ defmodule Tidewave.MCP.Tools.Source do
           }
         },
         callback: &get_source_location/1
+      },
+      %{
+        name: "get_package_location",
+        description: """
+        Returns the location of dependency packages.
+
+        You can use this tool to get the location of any project dependency. Optionally,
+        a specific dependency name can be provided to only return the location of that dependency.
+
+        Use the result in combination with shell tools like grep to look for specific
+        code inside dependencies.
+
+        Do NOT use `list_project_files` to inspect dependencies. It will NOT return files from dependencies,
+        but only files from the current project itself.
```

Comment:
Alternatively, if we want to keep it in the tool description:

```suggestion
        Packages that are placed with the current project will return
        a relative path and can be read as part of the project files.
        Dependencies with an absolute path must be read with care
        through shell commands.
```